usr/share/rear/format/USB/default/300_format_usb_disk.sh

```diff
@@ -58,36 +58,71 @@ else
     if ! parted -s $RAW_USB_DEVICE mklabel $USB_DEVICE_PARTED_LABEL >&2 ; then
         Error "Failed to create $USB_DEVICE_PARTED_LABEL partition table on '$RAW_USB_DEVICE'"
     fi
-    # Calculate byte value for the start of the subsequent ReaR data partition:
-    data_partition_start_byte=$(( USB_PARTITION_ALIGN_BLOCK_SIZE * MiB_bytes ))
+
     # rear_data_partition_number is used below and in the subsequent 350_label_usb_disk.sh script for the ReaR data partition:
     rear_data_partition_number=1
+
+    if [[ "$USB_DEVICE_PARTED_LABEL" == "gpt" ]] ; then
+        LogPrint "Creating bootloader system partition on '$RAW_USB_DEVICE' from sector 34 to 2047"
```

Comment:
I don't know the GPT specification in my mind.
Is a GPT bootloader system partition always from sector 34 to 2047?
I think the right naming for such a partition is `BIOS boot partition` according to
https://en.wikipedia.org/wiki/BIOS_boot_partition
so the LogPrint messsage should use that name to avoid confusion
what a "bootloader system partition" actually is.